package.json

```diff
@@ -52,12 +52,12 @@
     "nock": "^10.0.3",
     "spawn-mochas": "^2.0.0",
     "steal-stache": "^4.0.0",
-    "steal-tools": "^2.0.0-pre.3",
+    "steal-tools": "^2.0.0",
     "through2": "^2.0.1",
     "uglify-js": "^3.3.19"
   },
   "dependencies": {
-    "can-dom-mutate": "^1.2.1",
+    "can-dom-mutate": "^1.3.9",
```

Comment:
Did this need to be changed?

Reply:
`1.3.9` contains IE11 fixes.

Reply:
Maybe not as the rendering is done on the server.

Reply:
In general, we don’t need to increase the semver range when new versions are released, unless we _have to_ make sure the newer version is used.

Reply:
Fixed!